src/types.h

```diff
@@ -424,6 +424,20 @@ inline bool isElementTensorType(Type type) {
   return !isSystemTensorType(type);
 }
 
+inline bool isFixedSizeTensor(Type type) {
```

Comment:
This should probably be consolidated with `isElementTensorType` (they should be equivalent).